cobbler/modules/managers/import_signatures.py

```diff
@@ -119,7 +119,15 @@ def get_file_lines(self, filename: str) -> Union[List[str], List[bytes]]:
                     return file_fd.readlines()
             except Exception:
                 pass
-        if ftype.mime_type == "application/x-ms-wim":  # type: ignore
+        if (
+            ftype.mime_type == "application/x-ms-wim"  # type: ignore
+            or filename.lower().endswith(".wim")
+            and (
+                self.file_version[0] < 5
+                or self.file_version[0] == 5
+                and self.file_version[1] < 37
```

Comment:
In theory we can do this even shorter if desired:

```
>>> bool((5, 45) >= (5, 37))
True
>>> bool((5, 45) >= (5, 45))
True
>>> bool((5, 45) >= (5, 46))
```

Context: https://docs.python.org/3/reference/expressions.html#value-comparisons

Reply:
Thank you! This will really be better, I'll change the code.